editor/src/messages/tool/common_functionality/measure.rs

```diff
@@ -13,21 +13,41 @@ fn draw_dashed_line(line_start: DVec2, line_end: DVec2, transform: DAffine2, ove
 }
 
 /// Draws a solid line with a length annotation between two points transformed by the given affine transformations.
-fn draw_line_with_length(line_start: DVec2, line_end: DVec2, transform: DAffine2, document_to_viewport: DAffine2, overlay_context: &mut OverlayContext) {
+fn draw_line_with_length(
+	line_start: DVec2,
+	line_end: DVec2,
+	transform: DAffine2,
+	document_to_viewport: DAffine2,
+	overlay_context: &mut OverlayContext,
+	is_vertical_line: bool,
+	text_on_left: bool,
+	text_on_top: bool,
```

Comment:
Please refactor this into a struct instead of using three parameters.

Reply:
Sure! will a struct named something like `textPlacementParameters` do? i am assuming it's fine if the struct takes in the same 3 bool arguments as  values?

Reply:
Maybe something a bit shorter like `LabelAlignment`.

Reply:
I have pushed the code changes.